pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -418,19 +430,19 @@ func (vm *windows) waitForHNSNetworks() error {
 	return errors.Wrap(err, "timeout waiting for OVN overlay HNS networks")
 }
 
-// waitForHybridOverlayToRun waits for the hybrid-overlay-node.exe to run until the timeout is reached
-func (vm *windows) waitForHybridOverlayToRun() error {
+// waitForServiceToRun waits for the given service to be in RUNNING state
+// until the timeout is reached
+func (vm *windows) waitForServiceToRun(serviceName string) error {
```

Comment:
Shouldn't this be in the service restructuring commit?

Reply:
I have changed this method to accomodate for the service that I am adding, hence it made  sense to me to keep it in the feature commit.